orchestra/static/orchestra/common/js/orchestra_api.js

```diff
@@ -0,0 +1,62 @@
+var serviceModule =  angular.module('orchestra.common.services');
+
```

Comment:
high-level: make the url_endpoints and `methodNames` be the same (e.g., `edit_slack_membership` vs `editSlackGroupMembership`)